k6LoadTests.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable */
+import { sleep } from 'k6';
+import http from 'k6/http';
+
+export const options = {
+  duration: '10m',
+  vus: 20,
+  //  vus: 1,
+  thresholds: {
+    //avg is around ?800ms? on https://palvelutarjotin-api.test.kuva.hel.ninja
+    http_req_duration: ['p(95)<5000'],
+  },
+};
+
+export default () => {
```

Comment:
Would it make any sense to add this K6 to the docker-compose configuration? Installation instructions: https://k6.io/docs/getting-started/installation/#docker.